src/pages/lib/storage/move/q/platform/[platform].mdx

```diff
@@ -0,0 +1,8 @@
+export const meta = {
+  title: `Move files`,
+  description: `Learn more about how to move files using Amplify's storage category.`
+};
+
+import flutter0 from '/src/fragments/lib/storage/flutter/move.mdx';
```

Comment:
This comment is in the wrong spot but I cannot find the right file in this PR - the upload file page does not have a switcher for the APIs - the stable and dev preview options are listed right after each other.